doc-content/enterprise-only/openshift/environment-managed-deploy-proc.adoc

```diff
@@ -41,6 +41,7 @@ In this command line:
 ** *Smart Router Keystore Password* (`KIE_SERVER_ROUTER_HTTPS_PASSWORD`): The password for the keystore that you created in <<secrets-smartrouter-create-proc>>.
 ** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): The name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
 ** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): The password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Mode* (`KIE_SERVER_MODE`): In the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` template the default value is `PRODUCTION`; in this mode, you can not deploy `SNAPSHOT` versions of KJAR artifacts and can not change versions of an artifact in an existing container. (You can still create a new container on the same {KIE_SERVER} to deploy a new version). If you want to deploy `SNAPSHOT` versions or to change versions of an artifact in an existing container, set this parameter to `DEVELOPMENT`. 
```

Comment:
Minor rewrite suggestion:

Create a new container on the same {KIE_SERVER} to deploy a new version. To deploy `SNAPSHOT` versions or to change versions of an artifact in an existing container, set this parameter to `DEVELOPMENT`.